zerver/openapi/zulip.yaml

```diff
@@ -1419,10 +1434,22 @@ paths:
                                 [GET /streams](/api/get-streams#response) response
                                 for details on the various properties of a channel.
 
-                                **Changes**: Before Zulip 9.0 (feature level 256), this event
-                                was never sent when the `first_message_id` property of a channel
-                                was updated because the oldest message that had been sent to it
+                                This event is also sent when archiving or unarchiving a
+                                channel to all the users who can see that channel exists
+                                but only to the clients that declared the `archived_channels`
+                                [client capability][client-capabilities].
+
+                                **Changes**: Starting with Zulip 11.0 (feature level 376),
+                                this event is also sent when archiving or unarchiving
+                                channels to clients that declared the `archived_channels`
+                                [client capability][client-capabilities].
```

Comment:
I think it would be clearer to say "Prior to Zulip 11.0 (), this event was never sent when archiving/unarchiving a channel.", similar to the previous note.

Reply:
Updated.